Tests/Command/GenerateAdminCommandTest.php

```diff
@@ -125,14 +123,14 @@ public function testExecute()
         $command = $this->application->find('sonata:admin:generate');
         $commandTester = new CommandTester($command);
         $commandTester->execute(array(
-            'command' => $command->getName(),
-            'model'=>'Sonata\AdminBundle\Tests\Fixtures\Bundle\Entity\Foo',
-            '--bundle'=>'AcmeDemoBundle',
-            '--admin'=>'FooAdmin',
-            '--controller'=>'FooAdminController',
-            '--services'=>'admin.yml',
-            '--id'=>'acme_demo_admin.admin.foo',
-            ), array('interactive'=>false,));
+            'command'      => $command->getName(),
+            'model'        => 'Sonata\AdminBundle\Tests\Fixtures\Bundle\Entity\Foo',
+            '--bundle'     => 'AcmeDemoBundle',
+            '--admin'      => 'FooAdmin',
+            '--controller' => 'FooAdminController',
+            '--services'   => 'admin.yml',
+            '--id'         => 'acme_demo_admin.admin.foo',
+            ), array('interactive' => false));
```

Comment:
There is no option for this in the cs fixer ? how symfony manage to do that: https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/BrowserKit/Client.php#L115-L118